assets/js/frontend/checkout.js

```diff
@@ -271,6 +275,9 @@ jQuery( function( $ ) {
 				url:		wc_checkout_params.wc_ajax_url.toString().replace( '%%endpoint%%', 'update_order_review' ),
 				data:		data,
 				success:	function( data ) {
+
+					var selectedPaymentMethod = $( '.woocommerce-checkout' ).find( 'input[name="payment_method"]' ).filter( ':checked' ).attr( 'id' );
```

Comment:
Any performance improvement with `filter()` vs `:checked` in the first selector?